pkgs/tools/admin/aws-auth/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, makeWrapper, jq, awscli }:
+
+stdenv.mkDerivation rec {
+  version = "unstable-2017-07-24";
```

Comment:
That is our current policy per https://nixos.org/nixpkgs/manual/#sec-package-naming